app/views/home/index.html.slim

```diff
@@ -1,18 +1,18 @@
 html
```

Comment:
the html, head & body tags should be coming from the application.html.slim and should not be included in any of the other views